<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601915635831' stop='1601915655975'>
    <name>Add KNOS Relationship</name>
    <title>Add KNOS Relationship</title>
    <test-cases>
        <test-case start='1601915635831' status='passed' stop='1601915641729'>
            <name>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</name>
            <title>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='d02df0b1-c7db-490d-a511-02c18d1cb813-attachment.png' type='image/png' size='42550'/>
            </attachments>
        </test-case>
        <test-case start='1601915641729' status='passed' stop='1601915648600'>
            <name>AddNewKNOS- It should add new casetype and subtype releationship to system!</name>
            <title>AddNewKNOS- It should add new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='419bd7f8-711f-43ed-8488-94feeb10f566-attachment.png' type='image/png' size='42918'/>
            </attachments>
        </test-case>
        <test-case start='1601915648600' status='failed' stop='1601915655975'>
            <name>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</name>
            <title>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='f0c39f7d-8148-4930-9d62-34dbd31a37b0-attachment.png' type='image/png' size='58009'/>
            </attachments>
            <failure>
                <message>Expected 'Professional Tort' to equal 'CaseType123'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (C:\Users\UX013810\workspace\WCWRUI\Tests\Admin\UC_AD_16\AddNewKNOSTest.js:69:54)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2974:25
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>